src/models/state/ui.models.ts

```diff
@@ -31,12 +33,15 @@ export interface ITriggerFlashMessagePayload {
     navigateToRoute?: INavigateToRoute;
 }
 
-interface IListFilters<ColumnNames> {
-    filters: ListFilters<Partial<ColumnNames>>;
+interface IListFilters<T> {
+    filters: ListFilters<Partial<T>>;
     page: number;
-    sortedColumn: ISortedColumn<ColumnNames>;
+    sortedColumn: ISortedColumn<T>;
```

Comment:
why is this changed?

Reply:
Because the sorted column was only for "ColumnNames" types which has column names for script only. I created a generic sortedColum to take other column type